Lombiq.NodeJs.Extensions/scripts/handle-error.js

```diff
@@ -4,16 +4,19 @@
 
 const os = require('os');
 
-// Treat this dependency as optional because it's not available everywhere.
-let chalk;
-try {
-    chalk = require('chalk');
-}
-catch {
-    chalk = false;
+function handleColoring(output, type, errorColoring, warningColoring) {
+    let error = type === 'error';
+  
+    if (error && typeof errorColoring === 'function') {
+      return errorColoring(output);
```

Comment:
Indentation should be 4 spaces, not 2. Please change this in VS, if you're using that.